File: Java言語/基礎知識用の勉強/4繰り返し勉強/forの入れ子.java
class forの入れ子 {
	public static void main(String[] args){
		String asakunotomohiro = new String("朝来野智博");
		System.out.print("hello World.\n");

		int asakuno = 20210919;
		int tomohiro = 20210920;

		for ( int i = 0; 3 > i; i++ ) {
			for ( int j = 0; 3 > j; j++ ) {
				System.out.println( "i == " + i + ", j == " + j );	//
			}
		}
		// 出力結果：
//					i == 0, j == 0
//					i == 0, j == 1
//					i == 0, j == 2
//					i == 1, j == 0
//					i == 1, j == 1
//					i == 1, j == 2
//					i == 2, j == 0
//					i == 2, j == 1
//					i == 2, j == 2

	}
}

// vim: set ts=4 sts=4 sw=4 tw=0 ff=unix fenc=utf-8 ft=java noexpandtab:

File: Go言語/基礎知識用の勉強/4繰り返し勉強/実行済み/forの入れ子.go
package main

import "fmt"

func main() {
	asakuno := [...]int {20210929, 20210930 };
	tomohiro := [...]string {"asakuno", "tomohiro" };

	for ii, value := range asakuno {
		for jj, valuf := range tomohiro {
			fmt.Printf("ii：%d-jj：%d, asakuno[%d], tomohiro[%s]\n", ii, jj, value, valuf)
		}
	}
	// 出力結果：
//				ii：0-jj：0, asakuno[20210929], tomohiro[asakuno]
//				ii：0-jj：1, asakuno[20210929], tomohiro[tomohiro]
//				ii：1-jj：0, asakuno[20210930], tomohiro[asakuno]
//				ii：1-jj：1, asakuno[20210930], tomohiro[tomohiro]
	fmt.Println("-----------------------");
}

/* vim: set ts=4 sts=4 sw=4 tw=0 ff=unix fenc=utf-8 ft=go noexpandtab: */
